### ISSUE TITLE
user.present ignores "createhome: False"

### ISSUE THREAD

**Issue by feigenblatt** [saltstack/salt#28726]
salt-call --versions-report
```
Salt Version:
           Salt: 2015.8.1

Dependency Versions:
         Jinja2: 2.8
       M2Crypto: Not Installed
           Mako: 1.0.1
         PyYAML: 3.10
          PyZMQ: 14.6.0
         Python: 2.7.10 (default, Nov  5 2015, 10:48:05)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: 1.2.4
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist: centos 7.0.1406 Core
        machine: x86_64
        release: 3.10.0-123.20.1.el7.x86_64
         system: CentOS Linux 7.0.1406 Core
```

User home account directories are being created even when we set "createhome" to False.
In other words, the -M flag is not being passed to the "useradd" command and the directory "/home/someuser" will be created when using the following formula:

```
add_some_user:
  user.present:
    - name: someuser
    - createhome: False
```

**Comment by jfindlay** [comment]
@feigenblatt, thanks for the report.

**Comment by cachedout** [comment]
Fixed by #28758
